impeller/aiks/experimental_canvas.cc

```diff
@@ -514,16 +557,19 @@ bool ExperimentalCanvas::Restore() {
     // sampling, so aligning here is important for avoiding visual nearest
     // sampling errors caused by limited floating point precision when
     // straddling a half pixel boundary.
-    //
-    // We do this in lieu of expanding/rounding out the subpass coverage in
-    // order to keep the bounds wrapping consistently tight around subpass
-    // elements. Which is necessary to avoid intense flickering in cases
-    // where a subpass texture has a large blur filter with clamp sampling.
-    //
-    // See also this bug: https://github.com/flutter/flutter/issues/144213
-    Point subpass_texture_position =
-        (save_layer_state.coverage.GetOrigin() - GetGlobalPassPosition())
-            .Round();
+    Point subpass_texture_position;
+    if (transform_stack_.back().did_round_out) {
```

Comment:
@flar to double check my math here...

Reply:
I think rather than a flag it would be better to have the subpass record "I represent the pixels located at [rect]" in the long run, I think that's all the information you need and it doesn't require the back-half to know how the front-half did its math.